### ISSUE TITLE
[BUG] OneDriverGUI 2.5.2 does not sync

### ISSUE THREAD

**Issue by ilchimicopazzo** [bpozdena/OneDriveGUI#201]
**Describe the bug**
From some weeks, my OneDrive GUI (version 2.5.2) has shown the message «OneDrive sync is starting...» and a red light.
I am using the OneDrive GUI Appimage on updated Debian 12. I tried to lunch the appimage from terminal and I found the messages reported beyond.

**To Reproduce**
Steps to reproduce the behaviour:
1. Install OneDriveGUI Appimage version 2.5.2 following the instructions.
2. Install also dependencies.
3. Lunch the Appimage.

**Expected behavior**
OneDriveGui should start the sync, but it does not happen!

**Screenshots**
https://ibb.co/k58ZMc9

**Log**
I removed any personal information

2024-10-21 17:15:41,280 [OneDriveGUI.py:3075][fn=main_window_start_state][INFO] - [GUI] Starting OneDriveGUI maximized
2024-10-21 17:15:41,283 [OneDriveGUI.py:1108][fn=__init__][INFO] - name.surname@email.com
2024-10-21 17:15:41,797 [connectionpool.py:1051][fn=_new_conn][DEBUG] - Starting new HTTPS connection (1): api.github.com:443
2024-10-21 17:15:42,244 [connectionpool.py:546][fn=_make_request][DEBUG] - https://api.github.com:443 "GET /repos/abraunegg/onedrive/releases/latest HTTP/11" 200 1027
2024-10-21 17:15:42,281 [OneDriveGUI.py:2444][fn=client_version_check][DEBUG] - [GUI] Client version check: Installed: v2.5.2 | Latest: v2.5.2
2024-10-21 17:15:42,289 [OneDriveGUI.py:2526][fn=autostart_monitor][DEBUG] - [name.surname@email.com] Compatible client version found: True
2024-10-21 17:15:42,289 [OneDriveGUI.py:2529][fn=autostart_monitor][DEBUG] - [name.surname@email.com] Auto-sync enabled for profile: True
2024-10-21 17:15:42,289 [OneDriveGUI.py:1945][fn=__init__][INFO] - [GUI] Starting worker for profile name.surname@email.com
2024-10-21 17:15:42,289 [OneDriveGUI.py:1949][fn=__init__][DEBUG] - [GUI] OneDrive config file: /home/$USER/.config/onedrive/accounts/name.surname@email.com/config
2024-10-21 17:15:42,289 [OneDriveGUI.py:1950][fn=__init__][DEBUG] - [GUI] OneDrive config dir: <re.Match object; span=(0, 73), match='/home/$USER/.config/onedrive/accounts/andrea.>
2024-10-21 17:15:42,289 [OneDriveGUI.py:1952][fn=__init__][DEBUG] - [GUI] Monitoring command: 'exec onedrive --confdir='/home/$USER/.config/onedrive/accounts/name.surname@email.com' --monitor -v '
2024-10-21 17:15:42,290 [OneDriveGUI.py:2547][fn=start_onedrive_monitor][INFO] - [GUI] Opening external login window
2024-10-21 17:15:42,312 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Reading configuration file: /home/$USER/.config/onedrive/accounts/name.surname@email.com/config
2024-10-21 17:15:42,312 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] The option 'force_http_2' has been deprecated and will be ignored. Please read the updated documentation and update your client configuration to remove this option.
2024-10-21 17:15:42,312 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Configuration file successfully loaded
2024-10-21 17:15:42,312 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Using 'user' configuration path for application config and state data: /home/$USER/.config/onedrive/accounts/name.surname@email.com
2024-10-21 17:15:42,323 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] D-Bus message bus daemon is available; GUI notifications are now enabled
2024-10-21 17:15:42,328 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Using IPv4 and IPv6 (if configured) for all network operations
2024-10-21 17:15:42,350 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Attempting to contact Microsoft OneDrive Login Service
2024-10-21 17:15:42,576 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Successfully reached Microsoft OneDrive Login Service
2024-10-21 17:15:42,578 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Checking Application Version ...
2024-10-21 17:15:42,676 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Attempting to initialise the OneDrive API ...
2024-10-21 17:15:42,676 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Configuring Global Azure AD Endpoints
2024-10-21 17:15:42,676 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] The OneDrive API was initialised successfully
2024-10-21 17:15:42,677 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] Opening the item database ...
2024-10-21 17:15:42,678 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] The item database is incompatible, re-creating database table structures
2024-10-21 17:15:43,282 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] sqlite.SqliteException@src/sqlite.d(141): no such table: item
2024-10-21 17:15:43,282 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ----------------
2024-10-21 17:15:43,282 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? [0x55794c916617]
2024-10-21 17:15:43,283 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? [0x55794c919f08]
2024-10-21 17:15:43,283 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? [0x55794c9e45dc]
2024-10-21 17:15:43,283 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? void rt.dmain2._d_run_main2(char[][], ulong, extern (C) int function(char[][])*).runAll() [0x7fe6553bdb7b]
2024-10-21 17:15:43,283 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? _d_run_main2 [0x7fe6553bd995]
2024-10-21 17:15:43,283 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? _d_run_main [0x7fe6553bd7ed]
2024-10-21 17:15:43,284 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? [0x55794ca1a7a1]
2024-10-21 17:15:43,284 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? [0x7fe654b67249]
2024-10-21 17:15:43,284 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? __libc_start_main [0x7fe654b67304]
2024-10-21 17:15:43,284 [OneDriveGUI.py:2022][fn=read_stdout][INFO] - [name.surname@email.com] ??:? [0x55794c8ee220]
2024-10-21 17:15:44,287 [OneDriveGUI.py:2559][fn=<lambda>][INFO] - finished worker name.surname@email.com
2024-10-21 17:15:44,288 [OneDriveGUI.py:1956][fn=stop_worker][INFO] - [name.surname@email.com] Waiting for worker to finish...
2024-10-21 17:15:44,288 [OneDriveGUI.py:1960][fn=stop_worker][INFO] - [name.surname@email.com] Quitting thread
2024-10-21 17:15:44,288 [OneDriveGUI.py:1963][fn=stop_worker][INFO] - [name.surname@email.com] Removing thread info
2024-10-21 17:15:44,288 [OneDriveGUI.py:1966][fn=stop_worker][INFO] - [GUI] Remaining running workers: {}

**System Info**
 - Linux distribution: Debian 12
 - Desktop environment:  GNOME
 - Compositor: Wayland
 - Python version: 3.11.2
 - Version of OneDrive client: v2.5.2-1+np1+1.1

**OneDriveGUI info**
How did you install OneDriveGUI?: AppImage
What is the name of the AppImage file (if applicable)? : OneDriveGUI-1.1.0-x86_64.AppImage

**Comment by bpozdena** [comment]
It looks like onedrive client is crashing for some reason. 

Please run the client without the GUI using the below command : 

```
onedrive --confdir='/home/$USER/.config/onedrive/accounts/name.surname@email.com' --monitor -v
```

When it crashes, open an issue on the onedrive client GitHub page. 

Before doing so, maybe also try with `--resync` option. Or check if your files or directories contain some non English characters.

**Comment by abraunegg** [comment]
@bpozdena , @ilchimicopazzo 

An odd issue for sure.

The issue is that the application is detecting that the schema is not valid, but, when it goes to drop the table, the table does not exist thus crashes.

Please raise a bug here: https://github.com/abraunegg/onedrive for tracking purposes.

I have already identified the fix, will be included in v2.5.3

**Comment by ilchimicopazzo** [comment]
> @bpozdena , @ilchimicopazzo
> 
> An odd issue for sure.
> 
> The issue is that the application is detecting that the schema is not valid, but, when it goes to drop the table, the table does not exist thus crashes.
> 
> Please raise a bug here: https://github.com/abraunegg/onedrive for tracking purposes.
> 
> I have already identified the fix, will be included in v2.5.3

Thank you @abraunegg for your answer! I hope that the new version will be available soon.

Have a nice day!

**Comment by abraunegg** [comment]
Please raise the bug report .. the fix is already done
